recipe/meta.yaml

```diff
@@ -50,19 +50,19 @@ requirements:
     - importlib-metadata  # [py < 39]
 
   run_constrained:
-    - {{ pin_compatible('tbb', max_pin=None) }}
+    - {{ pin_compatible('tbb-devel', max_pin=None) | replace('-devel', '') }}
```

Comment:
Do we know why this was needed? Asking it seemed like the old constraint worked, but maybe I'm missing something

Reply:
Ah, I lumped this in with other changes.
This is not actually needed by a relic from local tests I had lying around.